tests/Integration/TransactionIntegrationTest.php

```diff
@@ -49,22 +49,25 @@ public function setupOnceDataProvider(): \Generator
             Event::createEvent(),
             true,
             [],
-            [],
             null,
         ];
 
         yield [
-            Event::createEvent(),
+            (static function () {
```

Comment:
If you move the instantiation of the event before the `yield` statement we can avoid creating all these self-invokable lambdas which makes the code less readable